lambda/textractor/python/og.py

```diff
@@ -212,13 +224,11 @@ def run(self):
             docText = docText + page.text + "\n"
 
             if(self.forms):
-                self._outputForm(page, p)
+                key_val_pairs = self._outputForm(page, p)
 
             if(self.tables):
                 self._outputTable(page, p)
 
             p = p + 1
 
-        return docText
-        # if(self.elasticsearchDomain):
-        #     self._indexDocument(docText)
+        return docText, key_val_pairs
```

Comment:
Can we return this as a dictionary rather than a tuple?
Depending upon if we need to add more data, it is more scalable and maintainable to have a dictionary with keys "docText", "KVPairs"

Reply:
Makes sense. I will do the changes.